### PR TITLE
Debian Package

### PR DESCRIPTION
**Pull request by andrewdeandrade**
@prashantv

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,9 +1,32 @@
-.PHONY: build install
+PREFIX=/usr/bin
 
-all: build
+PKG=github.com/yarpc/yab
```
Comment (akshayjshah) [inline Makefile]: update to tcheck?

File: Makefile [hunk 2]
```diff
@@ -1,9 +1,32 @@
...
-install:
-	cp tcheck ${DESTDIR}/local/bin/tcheck
+$(GOPATH):
+	mkdir -p $(GOPATH)/src
+	mkdir -p $(GOPATH)/pkg
+	mkdir -p $(BINDIR)
+
+$(BINDIR)/$(BINNAME): $(GOPATH)
+	git clone --depth 1 https://$(PKG) $(GOPATH)/src/$(PKG)
+	cp $(GOPATH)/src/$(PKG)/man/yab.1 debian/
```
Comment (akshayjshah) [inline Makefile]: update to tcheck

File: debian/copyright [hunk 3]
```diff
@@ -0,0 +1,15 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: uber-yab
```
Comment (akshayjshah) [inline debian/copyright]: update to tcheck.

**Comment by andrewdeandrade** [comment]
@akshayjshah sorry about that. That's what I get for trying to do both at the same time.

File: debian/control [hunk 4]
```diff
@@ -0,0 +1,12 @@
+Source: uber-tcheck
+Section: Utilities
+Priority: extra
+Maintainer: Andrew Andrade <aandrade@uber.com>
+Build-Depends: debhelper (>= 8.0.0), golang (>= 2:1.5.3-1), uber-glide, git, mercurial
```
Comment (andrewdeandrade) [inline debian/control:5]: is uber-glide a public debian package?

Comment (blampe) [inline debian/control:5]: Not that I'm aware of.

**Comment by blampe** [comment]
Can you help us understand the purpose of this change?

**Comment by andrewdeandrade** [comment]
Turns out our jenkins debian build process won't work for go projects so I'm closing this in favor of the debian build process we already use for yarpc/yab

File: debian/control (hunk 4 above)
Comment (bondareko) [inline debian/control]: Common Stock Ex-Distribution When Issued

**Pull request closed without merging**